tests/cmdline/commands/test_setup.py

```diff
@@ -79,8 +79,8 @@ def test_quicksetup(self):
         # Check that the repository UUID was stored in the database
         manager = get_manager()
         backend_manager = manager.get_backend_manager()
-        repository = profile.get_repository_container()
-        self.assertEqual(backend_manager.get_repository_uuid(), repository.container_id)
+        repository_uuid = profile.get_repository().uuid
+        self.assertEqual(backend_manager.get_repository_uuid(), repository_uuid)
```

Comment:
It reads a bit redundant now I think. But up to you.
```suggestion
        self.assertEqual(backend_manager.get_repository_uuid(), profile.get_repository().uuid)
```

Reply:
If it's used later as a proper variable fine to keep (can't see that it is). The same goes for the suggested change below.